src/content/en/2020/markup.md

```diff
@@ -0,0 +1,626 @@
+featured_stat_1: 85.73%
+featured_stat_label_1: Percentage of pages that use the “living” HTML doctype
+featured_stat_2: 30,073
+featured_stat_label_2: Number of non-standard `h7` elements
+featured_stat_3: 25.24 KB
+featured_stat_label_3: Weight of the median document
+published: 2020-11-01T00:00:00.000Z
+last_updated: 2020-11-04T00:00:00.000Z
+---
+
+## Introduction
+
+The Web is built on HTML. Without HTML: no web pages, no web sites, no web apps. Nothing. Plain-text documents, perhaps, or XML trees, in some parallel universe that enjoyed that particular kind of challenge. In this universe, HTML is the foundation of the user-facing Web. There are many standards that the Web is resting on, but HTML is certainly one of the most important ones.
+
+How do we use HTML, then, how great of a foundation are we looking at (and in this Markup chapter, we’re focusing almost exclusively on HTML)? The introduction of [last year’s markup chapter](../2019/markup#introduction) suggested that for a long time, we haven’t really known: There were some smaller samples, there was Ian Hickson’s, one of modern HTML’s parents, research [back in 2005](https://web.archive.org/web/20060203035414/http://code.google.com/webstats/index.html), then there were more small samples, but until last year we lacked major insight into how we as developers, as authors, make use of HTML. In 2019 we then had both [Catalin Rosu’s work](https://www.advancedwebranking.com/html/) (one of this chapter’s co-authors) as well as the 2019 edition of the Web Almanac to give a better view again, at HTML in practice.
+
+Last year marked a first stab at 5,790,700 pages (of which 4,371,973 were tested on desktop, 5,297,442 on mobile) and a review of the data taken. This year we looked at 7,546,709 pages (5,593,642 on desktop, 6,347,919 on mobile) using the latest data on [which websites users are visiting in 2020](./methodology), to run another analysis. We do make some comparisons to last year—but, just as we’ve tried to take some additional metrics for additional insight, we’ve also given this chapter a new personality.
+
+<p class="note">The data for all Almanac chapters is open and available. Take a look at [the results](https://docs.google.com/spreadsheets/d/1Ta7amoUeaL4pILhWzH-SCzMX9PsZeb1x_mwrX2C4eY8/) and share your own observations!</p>
+
+## General
+
+In this section we’re having a look at higher-level aspects of HTML, like document types, the size of documents, as well as the use of comments and scripts. “Living HTML” is very much alive!
+
+### Doctypes
+
+96.99% of the sampled pages use a doctype. That is useful, because, for historic reasons (“[to avoid triggering quirks mode](https://lists.w3.org/Archives/Public/public-html-comments/2009Jul/0020.html) in browsers”) HTML documents need to declare a document type.
```

Comment:
Again more for editing, but will mark here incase you have any strong reservations against these sorts of changes @j9t 

```suggestion
96.99% of the sampled pages use a doctype. That is useful because, for historic reasons ([to avoid triggering quirks mode](https://lists.w3.org/Archives/Public/public-html-comments/2009Jul/0020.html) in browsers) HTML documents need to declare a document type.
```

Reply:
I’m going to revisit the use of commas here but like to keep the quotes, because it is a quote.